cloudbuild.yaml

```diff
@@ -51,20 +51,28 @@ steps:
     - '--key=$_CPP_SDK_CONFORMANCE_CACHE_KEY-$( checksum sdks/cpp/cmake/prerequisites.cmake )'
   waitFor: ['-']
 
-
 - name: gcr.io/$PROJECT_ID/restore_cache
   id: rust-sdk-build-restore-cache
   args:
     - '--bucket=gs://$_CACHE_BUCKET'
     - '--key=$_RUST_SDK_BUILD_CACHE_KEY-$( checksum test/sdk/rust/Cargo.toml )'
   waitFor: ['-']
 
+#
+# Enable binfmt 
+# https://docs.docker.com/buildx/working-with-buildx/
+#
+
+- name: 'gcr.io/cloud-builders/docker:20.10.3'
```

Comment:
So fun thought on this step - should this be included in the makefile?

If I'm running any of these commands locally, and I want to create a local arm image,   

I'm thinking an `ensure-arm-builder` target, and make it a dependency of the following targets: `build-controller-image build-agones-sdk-image build-ping-image build-allocator-image` ?

Then we don't have to add this to the cloudbuild.yaml and it works the same local as it does in CI - making things easier to replicate.

WDYT?

Reply:
Yeah, it's a good idea !